test/jdk/java/awt/FullScreen/FullscreenWindowProps/FullscreenWindowProps.java

```diff
@@ -52,6 +52,10 @@ public void paint(Graphics g) {
                 super.paint(g);
                 g.setColor(Color.GREEN);
                 g.fillRect(0, 0, getWidth(), getHeight());
+                g.setColor(Color.RED);
+                DisplayMode displayMode =
+                        getGraphicsConfiguration().getDevice().getDisplayMode();
+                g.drawString(displayMode.toString(), 100, 100);
             }
```

Comment:
@azvegint 
This is on an unchanged line. The finally block in the test sets `gd.setFullScreenWindow(null);`
Is that sufficient or do we need to additionally restore the original display mode in the finally block?